WordPress/src/main/res/layout/prepublishing_home_compose_item.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+    <View
+        android:id="@+id/top_divider"
+        android:layout_width="match_parent"
```

Comment:
💡 Would it make sense to use `0dp` instead of `match_parent` and add `start` and `end` constraints?

Reply:
Good point, done!